backend/src/main/java/com/group/backend/domain/PortalRepository.java

```diff
@@ -1,11 +1,14 @@
 package com.group.backend.domain;
 
+import com.group.backend.entity.Portal;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
-import com.group.backend.entity.Portal;
-import java.util.List; // Certifique-se de importar java.util.List
 
 @Repository
 public interface PortalRepository extends JpaRepository<Portal, Long> {
```

Comment:
o yan conseguiu resolver isso com um PortalTagRepository, que é a forma mais correta de você iteragir quando vc tem uma tabela many to many, não quer dar uma olhada nisso?